packages/react-vapor/src/components/button/examples/ButtonExamples.1.use.md

```diff
@@ -0,0 +1,39 @@
+## Guidelines
+
+### General
+
+-   To redirect users to another page of the current website or app, or to another website, use links rather than buttons.
+-   Avoid multiplying buttons within a given page. If you need several buttons, less popular or less important actions may be grouped in a “more” button (dropdown list) or visually styled as isolated actions or links.
```

Comment:
I'm not sure what you mean by visually styled as isolated actions or links though? Maybe you could try to rephrase, and I can suggest something :)